src/ProjectTemplates/Web.ProjectTemplates/content/WebApi-CSharp/Controllers/WeatherForecastController.cs

```diff
@@ -54,7 +54,17 @@ public async Task<IEnumerable<WeatherForecast>> Get()
         {
             HttpContext.VerifyUserHasAnyAcceptedScope(scopeRequiredByApi);
 
-            string downstreamApiResult = await _downstreamWebApi.CallWebApiAsync();
+            var response = await _downstreamWebApi.CallWebApiForUserAsync("DownstreamApi").ConfigureAwait(false);
```

Comment:
Here and elsewhere we can structure the requests using:

```
using (var response = await _downstreamApi.CallWebApiForUserAsync...) {
   ....
}
```

Reply:
Or just
```
using var response = await _downstreamWebApi.CallWebApiForUserAsync("DownstreamApi").ConfigureAwait(false);
```